.github/workflows/fuzz.yml

```diff
@@ -21,11 +21,8 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install --upgrade coverage
-          python -m pip install --upgrade hypothesmith
-          python -m pip install -e ".[d]"
+          python -m pip install --upgrade tox
```

Comment:
Why not install together?
`python -m pip install --upgrade pip tox`

Reply:
Maybe the newest pip has something new and exciting, and we want to use the new and exciting pip for the other stuff?

Reply:
Just seems a waste of an extra fork/exec for now ... If we hit a bug then I'd change back to the two executes ... Not a big deal, this why I didn't request changes ...

Reply:
I'm with @hugovk on this.

Reply:
If it were a single call to pip, then there's no point upgrading pip (unless it is cached for _next time_, and it's not).

A concrete reason:

For Python 3.8 on Windows, pip 19.3+ is needed to install wheels (at least for Pillow). We've added the double command to the [installation instructions](https://pillow.readthedocs.io/en/stable/installation.html#basic-installation), and that came up again [this week](https://github.com/python-pillow/Pillow/issues/4970#issuecomment-707299998).

This isn't _specifically_ needed for Black test right now, but I suggest skipping the pip upgrade altogether and using the CI image's version, or having two calls.

(Right now, pip in the image is the latest version, so the call is really quick; and it's pretty quick for an upgrade as well.)